docs/resources/Guild.md

```diff
@@ -651,6 +652,26 @@ In guilds with [Membership Screening](https://support.discord.com/hc/en-us/artic
 > warn
 > We are making significant changes to the Membership Screening API specifically related to getting and editing the Membership Screening object. Long story short is that it can be improved. As such, we have removed those documentation. There will **not be** any changes to how pending members work, as outlined above. That behavior will stay the same.
 
+### Incidents Data Object
+
+###### Incidents Data Structure
+
+| Field                  | Type               | Description                            |
+|------------------------|--------------------|----------------------------------------|
+| invites_disabled_until | ?ISO8601 timestamp | when invites get enabled again         |
+| dms_disabled_until     | ?ISO8601 timestamp | when direct messages get enabled again |
+| dm_spam_detected_at    | ?ISO8601 timestamp | when the dm spam was detected at       |
+| raid_detected_at       | ?ISO8601 timestamp | when the raid was detected at          |
+
+###### Example Incidents Data
+
+```json
+{
+  "invites_disabled_until": "2023-09-01T14:48:02.222000+00:00",
+  "dms_disabled_until": null
+}
+```
```

Comment:
Either the example is incomplete or `dm_spam_detected_at` and `raid_detected_at` should be marked as optional in the above table.